### DISCUSSION TITLE
Install Stuck With No Error

### DISCUSSION THREAD

**Discussion by JamConnor** [UltimaHoarder/UltimaScraper#1777] [Q&A]
I am using Python 3.9. I install everything in the requirements folder fine but when I try to run "python start_ofd.py" nothing happens.

![image](https://user-images.githubusercontent.com/95383409/144314063-2c72ca62-114c-4d07-b81d-cbcb4f6c49a1.png)

Any ideas

**Comment by drasticcode** [comment]
[Python error: String index out of range python](https://drasticcode.com/string-index-out-range-python/)

What is the “string index out of range” error, why it occurs, the solution to remove the string index out of range error

**Reply by fightcancer2** [reply]
If that didn't work, uninstall all Python versions and then install 3.10.1.
